packages/nice_parser/nice_parser.1.0.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["Steffen Smolka <smolka@cs.cornell.edu>"]
+authors: ["Steffen Smolka <smolka@cs.cornell.edu>"]
+bug-reports: "https://github.com/smolkaj/ocaml-parsing/issues"
+homepage: "https://github.com/smolkaj/ocaml-parsing"
+doc: "https://smolkaj.github.io/ocaml-parsing/nice_parser/"
+license: "MIT"
+dev-repo: "git+https://github.com/smolkaj/ocaml-parsing.git"
+synopsis: "Nice parsers without the boilerplate"
+description:
+  "Nice_parser wraps your {menhir, ocamlyacc}-generated parser in a sane interface, eliminating boilerplate code."
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "dune" {>= "1.10"}
+  "menhir" {build}
```

Comment:
```suggestion
  "menhir" {with-test}
```
menhir is not needed when compiling. Once the above comment is fixed it might be used for tests though